backend/infrahub/graphql/schema.py

```diff
@@ -81,6 +82,8 @@ async def account_resolver(root, info: GraphQLResolveInfo):
 
 
 class InfrahubBaseQuery(ObjectType):
+    SearchAnywhere = SearchAnywhere
```

Comment:
Even if it might look a bit excessive I wonder if we should name the public query `InfrahubSearchAnywhere` the reasoning here is that with the name `SearchAnywhere` it's possible for user to create a namespace called Search and a node called Anywhere. I'm not sure what would happen at that point but I'd guess we either get a failure or one of them are overwritten.

The two workarounds would be to add `Search` as a restricted namespace, or to move the query to a namespace that already is restricted.